app/controllers/user/shares_controller.rb

```diff
@@ -30,14 +30,7 @@ def new
   def create
     require_valid_params
 
-    birth_record = current_user.birth_records.find(share_params['birth_record_id'].to_i)
```

Comment:
This was mad about ABC score.
These assignments are unnecessary as they're only used once - but the score was still 16 after moving them down